javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,65 @@
+const notes = [];
+
+// implementing SAVE note
+function saveNote(content, id) {
+    const noteObject = { content: content, id: id };
+    notes.push(noteObject);
+}
+//check SAVE functionality
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+// implementing GET note
+function getNote(id) {
+    //log error in case id is not a number
+    if (typeof id !== 'number') {
```

Comment:
Well done to check your code for unintended values, but the same check is not applied to the save function, which is the arguably more important of the two.

In a real life scenario, this would mean that an item would be shown on the total overview of items, but when you then tried to retrieve the note item, it would be inaccessible if you previously saved a note with a string value as an id. The getter would fail when attempting to get the item with a string id, albeit being saved to the notes 'db'.

Reply:
Added checks